native/effektio-test/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 #[cfg(test)]
 mod auth;
-#[cfg(test)]
-mod emoji_verification;
+// #[cfg(test)]
```

Comment:
What's the point of this? Doesn't the test work anymore? Can we please leave is enabled led?

Reply:
As I said, when `emoji_verification` test is enabled, `typing_notification` test is not working.
I guess frequent usage of registered accounts of synapse server has some problem.
When performing `typing_notification` test, should I make new account to synapse server?

Reply:
That's weird. Let's merge as is, but file a bug - this needs fixing, synapse should be able to deal with both.

Reply:
Just opened #202 and assigned it to you and me, because it is rust bug.